app/src/main/java/org/mozilla/fenix/settings/address/ext/Address.kt

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.address.ext
+
+import androidx.annotation.VisibleForTesting
+import mozilla.components.concept.storage.Address
+
+/**
+ * Generate a label item text for an [Address]. The combination of names is based on desktop code
+ * found here:
+ * https://searchfox.org/mozilla-central/rev/d989c65584ded72c2de85cb40bede7ac2f176387/toolkit/components/formautofill/FormAutofillNameUtils.jsm#400
+ */
+fun Address.getAddressLabel(): String = listOf(givenName, additionalName, familyName)
```

Comment:
```suggestion
fun Address.getFullName(): String = listOf(givenName, additionalName, familyName)
```

I am gonna sound like a broken record here because I have said this from the first review, but what I still want for our extension function naming here is to avoid tying the naming to the UI element. Semantically, this is just returning the full name. The reasoning I am very persistent on this naming is because most (if not all) of our extension functions don't really prescribe to UI element labelling, but towards what is actually is.

I am also anticipating we will be using this in the select address prompt, but there's always the possibility that we use these extension functions beyond just for the list item UI. Having a full name field in the data structure is very much also a possibility when you look beyond https://searchfox.org/mozilla-central/source/mobile/android/geckoview/src/main/java/org/mozilla/geckoview/Autocomplete.java#465

Reply:
Oh that's a good point. I was a little confused because the suggestion will ultimately mean this line in `AddressList`:
`description = address.getAddressLabel()`, but now that you've explained it I see what you're going for. Thanks for your patience on this point, I appreciate the explanation!